### ISSUE TITLE
Fix for multy: line 1: /multy/multy-back: not found

### ISSUE THREAD

**Issue by abitrolly** [Multy-io/Multy-back#124]
This image is built and deployed successfully, but fail to run:

https://github.com/Multy-io/Multy-back/blob/822857b7423583c113271ab7f4d4c989439f9035/Dockerfile#L24-L27

```
$ docker run multyio/multy-back
/bin/sh: /multy/multy-back: not found
```

Because:

https://stackoverflow.com/questions/34729748/installed-go-binary-not-found-in-path-on-alpine-linux-docker

CC @Enmk

**Comment by abitrolly** [comment]
Need to compile with `musl`, which is the default replacement for `glibc` on Apline.

**Comment by Enmk** [comment]
Fixed in 'dev' branch with PR #125
